### ISSUE TITLE
Create automatic build script for Android App

### ISSUE THREAD

**Issue by niccokunzmann** [fossasia/flappy-svg#250]
It is possible

- [ ] To build the app with gradle
- [ ] To write down the shell command for this
- [ ] To make travis execute the shell command and build the app when a commit occurs
- [ ] To upload the apk file automatically to github releases
- [ ] Link the releases in the flappy game

**Comment by Heredroky** [comment]
@niccokunzmann The app has to update only when someone creates a release?

**Comment by niccokunzmann** [comment]
I do not care. Let's see what works.

**Comment by Heredroky** [comment]
@niccokunzmann I will work on this, can you create a blank branch named "apk"?

**Comment by niccokunzmann** [comment]
Please do it in your own branch. If you have some work, we can put it in an other branch.

**Comment by niccokunzmann** [comment]
In the master branch, we could create a weekly build with travis, too.

**Comment by Heredroky** [comment]
@niccokunzmann I will make it update when someone make release

**Comment by niccokunzmann** [comment]
ok :)

**Comment by niccokunzmann** [comment]
@Heredroky Would you like to try out github releases?
Have a look [at the code](https://github.com/fossasia/meilix/blob/master/.travis.yml#L17) and at the [result](https://github.com/fossasia/meilix/releases).

**Comment by niccokunzmann** [comment]
@Heredroky I am waiting for the email of the build bot to arrive. The build bot account can have all the permissions in the token. https://github.com/flappysvgbuildbot

**Comment by Abhi2424shek** [comment]
@niccokunzmann any updates?

**Comment by niccokunzmann** [comment]
Yes, the build bot is there. I can generate such a token. However, I do not have the rights to add this token to travis. I can not edit the travis settings.

**Comment by niccokunzmann** [comment]
Somehow I lost the administrative privileges for the repository. I will see if I can regain them.

**Comment by Abhi2424shek** [comment]
@niccokunzmann i don't have administrative privileges but having write access i could help you, mail me the token or #slack and I will set it up if you want :)

**Comment by niccokunzmann** [comment]
@Abhi2424shek, I sent you an email. Thanks!

**Comment by niccokunzmann** [comment]
The user is [flappysvgbuildbot](https://github.com/flappysvgbuildbot). I do not see if it can contribute.

**Comment by Abhi2424shek** [comment]
I added the token as BOT_TOKEN

**Comment by Abhi2424shek** [comment]
Add it as a contributor now
